src/client.js

```diff
@@ -1185,6 +1185,25 @@ MatrixClient.prototype.checkEventSenderTrust = async function(event) {
     return await this._crypto.checkDeviceTrust(event.getSender(), device.deviceId);
 };
 
+/**
+ * Get information about the encryption of an event
```

Comment:
Should `checkEventSenderTrust` be removed here? The React SDK PR removes the only use of it, and it was marked unstable for cross-signing, so could be removed safely I think.